frontends/election-manager/src/components/export_ballot_pdfs_button.test.tsx

```diff
@@ -0,0 +1,176 @@
+import { fireEvent, waitFor } from '@testing-library/react';
+import { fakeLogger, LogEventId } from '@votingworks/logging';
+import { fakeKiosk, fakeUsbDrive } from '@votingworks/test-utils';
+import { assert, usbstick } from '@votingworks/utils';
+import React from 'react';
+import { fakeFileWriter } from '../../test/helpers/fake_file_writer';
+import { renderInAppContext } from '../../test/render_in_app_context';
+import { ExportBallotPdfsButton } from './export_ballot_pdfs_button';
+
+const { UsbDriveStatus } = usbstick;
+
+jest.mock('../components/hand_marked_paper_ballot');
+
+beforeEach(() => {
+  const mockKiosk = fakeKiosk();
+  mockKiosk.getUsbDrives.mockResolvedValue([fakeUsbDrive()]);
+  const fileWriter = fakeFileWriter();
+  mockKiosk.saveAs = jest.fn().mockResolvedValue(fileWriter);
+  mockKiosk.writeFile = jest.fn().mockResolvedValue(fileWriter);
+  window.kiosk = mockKiosk;
+});
+
+afterEach(() => {
+  delete window.kiosk;
+});
+
+test('button renders properly when not clicked', () => {
+  const { queryByText, queryByTestId } = renderInAppContext(
+    <ExportBallotPdfsButton />
+  );
+
+  expect(queryByText('Export Ballot PDFs')).toHaveProperty('type', 'button');
+  expect(queryByTestId('modal')).toBeNull();
+});
+
+test('modal renders insert usb screen appropriately', async () => {
+  const usbStatuses = [
+    UsbDriveStatus.absent,
+    UsbDriveStatus.recentlyEjected,
+    UsbDriveStatus.notavailable,
+  ];
+
+  for (const usbStatus of usbStatuses) {
+    const { unmount, getByText, queryAllByAltText, queryAllByTestId } =
```

Comment:
Instead of using the query methods returned by the `render` call, it's recommended to use the general query methods on the `screen` object ([relevant link](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen))